src/app/child-dev-project/children/child-photo-service/children-migration.service.ts

```diff
@@ -0,0 +1,26 @@
+import { Injectable } from "@angular/core";
+import { Database } from "../../../core/database/database";
+import { Child } from "../model/child";
+
+@Injectable({
+  providedIn: "root",
+})
+export class ChildrenMigrationService {
+  constructor(private database: Database) {}
+
+  async migratePhotoFormat(): Promise<void> {
```

Comment:
as this is only a simple renaming: shall we leave the old name to avoid migration? This can potentially create a number of conflicts if not all users' local data have been synced just before the migration is executed

Reply:
Yes the migration is not totally necessary, but without it we have to leave the fallback in the photo datatype for ever because we never know if all of the organizations have the new format.